homework-1.rb

```diff
@@ -1,6 +1,6 @@
 item = ARGV[0]
 price = ARGV[1].to_f/1.0
```

Comment:
If `ARGV[1]` is equal to `1` and you type cast it to float. It turns to `1.0`. What is `1.0/1.0`? 😂. The `/1.0` is redundant.